OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -32,13 +32,16 @@ public class ExplodesInfo : ITraitInfo, IRulesetLoaded
 		[Desc("Chance that this actor will explode at all.")]
 		public readonly int Chance = 100;
 
+		[Desc("Health level at which actor will explode.")]
+		public readonly int HealthPercentage = 0;
```

Comment:
I suggest changing this to `DamageThreshold` to match `DamagedWithoutFoundation`.